enterprise/templates/enterprise/grant_data_sharing_permissions.html

```diff
@@ -1,73 +1,84 @@
-                            name="data_sharing_consent"
-                            class="input-block checkbox"
-                            data-consent="${data_sharing_consent}"
-                            data-warning-message="${ messages['warning'] }"
-                        />
-                        <label for="register-data_sharing_consent">
-                            % if not course_specific:
-                                ${_("I agree to allow {platform_name} to share data about my enrollment, completion and performance in all {platform_name} courses and programs where my enrollment is sponsored by {provider}.*").format(provider=enterprise_customer_name, platform_name=platform_name)}
-                            % else:
-                                ${_("I agree to allow {platform_name} to share data about my enrollment, completion and performance in {course_name} with {provider}.*").format(provider=enterprise_customer_name, course_name=course_name, platform_name=platform_name)}
-                            % endif
-                        </label>
-                    </div>
-                    <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }" />
-                    % if course_specific and course_id:
-                        <input type="hidden" name="course_id" value="${ course_id }" />
-                    % endif
-                    % if redirect_url:
-                        <input type="hidden" name="redirect_url" value="${redirect_url}" />
-                    % endif
-                    % if enrollment_deferred:
-                        <input type="hidden" name="enrollment_deferred" value="true" />
-                    % endif
-                    <button type="submit" class="action-primary action">Submit</button>
-                    <p class="note">*${ messages['note'] }
-                </form>
-            </div>
-        </section>
-    </div>
-</main>
+  <%static:css group='style-vendor'/>
+    <%! main_css = "style-main-v2" %>
+<%static:css group='${self.attr.main_css}'/>
+  <%!
+    from django.utils.translation import ugettext as _
+    %>
+<html>
+  <font face="sans-serif">
+    <title>${ title_bar_prefix } | ${ platform_name }</title>
+    <head>
+      <link rel="stylesheet" href="${static.url('css/vendor/font-awesome.css')}"/>
+      <link rel="stylesheet" href="${static.url('enterprise/grant_data_sharing_permissions.css')}"/>
+      <meta name="viewport" content="user-scalable = no, width=device-width, initial-scale=1"/>
+      <script type="text/javascript" src="${static.url('common/js/vendor/jquery.js')}"></script>
+      <script type="text/javascript" src="${static.url('js/vendor/jquery.cookie.js')}"></script>
+      <script type="text/javascript" src="${static.url('enterprise/grant_data_sharing_permissions.js')}"></script>
+    </head>
+    <body>
+      <div class="logo-container">
+        <img src="${static.url('images/logo.png')}" alt=""/>
```

Comment:
Good to know. I'm going to leave this as not a link, but add the alt text - in general, the way we've been going with this page is to avoid links to the outside world.